css/main.scss

```diff
@@ -49,7 +49,6 @@
 @import 'recording';
 @import 'login_menu';
 @import 'popover';
-@import 'jitsi_popover';
```

Comment:
All JitsiPopover references should be removed at this point as it has been replaced with InlineDialog.